facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -139,7 +139,7 @@ class FaciaPicker extends GuLogging {
   def getTier(faciaPage: PressedPage)(implicit request: RequestHeader): RenderType = {
     lazy val participatingInTest = ActiveExperiments.isParticipating(DCRFronts)
     lazy val checks = dcrChecks(faciaPage)
-    lazy val dcrCouldRender = checks.values.forall(checkValue => checkValue == true)
+    lazy val dcrCouldRender = checks.values.forall(checkValue => checkValue)
```

Comment:
Keep it consistent with above.

```suggestion
    lazy val dcrCouldRender = checks.values.forall(identity)
```

Reply:
We use both across the place - I can't find good reasoning for either and this is what my IDE suggested.